nautobot/dcim/models/device_components.py

```diff
@@ -501,6 +504,15 @@ class Meta:
 
     def save(self, *args, **kwargs):
 
+        # set interface status to active if status not provided
+        if not self.status:
+            query = Status.objects.get_for_model(self)
+            try:
+                status = query.get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
+            except Status.DoesNotExist:
+                status = query.first()
```

Comment:
Same thought here - changing my mind from earlier, I now think it's probably better if we bubble up an appropriate exception if STATUS_ACTIVE doesn't exist, rather than picking an arbitrary/random status to use.